src/components/ha-hls-player.ts

```diff
@@ -34,8 +35,7 @@ class HaHLSPlayer extends LitElement {
   @property({ type: Boolean, attribute: "playsinline" })
   public playsInline = false;
 
-  @property({ type: Boolean, attribute: "allow-exoplayer" })
-  public allowExoPlayer = false;
```

Comment:
Why this change? You can't use an attribute now to set the property?

Reply:
It seemed like the attribute was not necessary and it works fine without it. As I commented before I'm not sure it was even working before as there is no attribute `allow-exoplayer` coming down from `ha-camera-stream` (`more-info-camera` passes down `allow-exoplayer` to `ha-camera-stream` as an attribute, but `ha-camera-stream` passes down `allowExoPlayer` to `ha-hls-player` as a property, so looking for the `allow-exoplayer` attribute in `ha-hls-player` comes up false. I'm a JS novice so maybe I'm missing something with the property to attribute relationship, but if it works this way isn't it more parsimonious?

Reply:
No, it limits the options and people expect it to be available as an attribute.

It will just work if you pass it as a property instead of an attribute.

Reply:
I see, so better to have it available as an attribute in case someone wants to access the `ha-hls-player `directly and not through `ha-camera-stream`?